robot_interface:
  base:
    plugin: SendFollowJointTrajectory
    namespace: /whole_body_controller/base/follow_joint_trajectory
    state_topic: /whole_body_controller/base/state
    fill_velocity_values: True
    robot_name: pr2
  body:
    plugin: SendFollowJointTrajectory
    namespace: /whole_body_controller/body/follow_joint_trajectory
    state_topic: /whole_body_controller/body/state
    fill_velocity_values: False
    robot_name: pr2